addons/zotero/models.py

```diff
@@ -216,3 +212,89 @@ def clear_settings(self):
         """Clears selected folder and selected library configuration"""
         self.list_id = None
         self.library_id = None
+
+    def serialize_folder(self, kind, id, name, path, parent=None, provider_list_id=None):
+        return {
+            'addon': 'zotero',
+            'kind': kind,
+            'id': id,
+            'name': name,
+            'path': path,
+            'parent_list_id': parent,
+            'provider_list_id': provider_list_id or id
+        }
+
+    def get_folders(self, path=None, folder_id=None, **kwargs):
+        """
+        Returns Zotero folders at any level.
+        Top level are group/personal libraries.  Secondary levels are folders within those libraries.
+        If path (also known as library id) is specified, then folders are returned from that specific library.
+
+        If no path(library) specified, then all libraries are returned.
+        """
+        if self.has_auth:
+            if path:
+                return self.get_sub_folders(path, folder_id)
+            else:
+                return self.get_top_level_folders(**kwargs)
+        else:
+            raise exceptions.InvalidAuthError()
+
+    def get_top_level_folders(self, **kwargs):
+        """
+        Returns serialized group libraries - your personal library along with any group libraries.
+
+        This is the top-tier of "folders" in Zotero.
+
+        You can use kwargs to refine what data is returned -  how to limit the number of group libraries,
+        whether to return the personal library alongside group_libraries, or append the total library count.
+        """
+        # These kwargs are passed in from ZoteroViews > library_list
+        limit = kwargs.get('limit', None)
+        start = kwargs.get('start', None)
+        return_count = is_truthy(kwargs.get('return_count', False))
```

Comment:
Minor: Parsing out truthy values is the responsibility of the view layer (which deals with HTTP requests)--not the model. In other words, `is_truthy` should be used here: https://github.com/CenterForOpenScience/osf.io/pull/8230/files#diff-5457c01b22e7fb7919080abd304e7f59R58

Reply:
Oh, I see, thanks!